perun-core/src/main/java/cz/metacentrum/perun/core/bl/TasksManagerBl.java

```diff
@@ -150,6 +150,8 @@
 	 */
 	List<Task> listAllTasksForFacility(int facilityId);
 
+	List<Task> listAllTasksForService(int serviceId);
```

Comment:
Please, add also a javadoc for this method.

Reply:
Added. TasksManagerBl appears to be in very sorry state.